_posts/2020-11-08-basic-coding-applications-for-commercial-strategy.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title:  "Basic Coding Applications for Commercial Strategy"
+date:   2020-11-08 12:54:04 -0700
+categories: sales commercial strategy revenue cogs margins applications
+---
+By, Debbie Vo
+
+Every department for every company can benefit from data analysis. Coding allows large data sets to be manipulated and presented in a helpful way. For all code, the goal is that you start with raw information and you end up with useful information derived from the raw form. In genetics, you may have genome sequences as your starting material that are used to determine the similarities and differences between the DNA for different samples. This analysis has many different applications, including the determination of mutations in cancer cells.
+
+Benefits of programming can also be found outside of research and development such as in sales and marketing. Below are a few examples of how R  code can be used to create impactful tools for commercial departments.
+
+
+# Data Cleaning
+Before we dive into each code example, it is crucial to clean your raw data first. The most important items to check are that the column titles are in line with what your code references as well as to check that the data points are consistently and correctly formatted.
+1. **Check raw data column titles:** The raw data column titles must match the code exactly including capitalization. With code that references `revenue`, a column title of `Revenue` will not work. If your general code references `territory`, but your raw data uses `states` as the territory descriptor, the column title for the states must be changed to `territory`.
+2. **Check data format consistency:** If a data point doesn’t exist, it needs to say `NA`. Otherwise, the data should follow the same template throughout the data set and that aligns with the code. For example, the state of California should not be represented as `California`, `california`, `CA`, and `ca`. It doesn't matter which one is used as long as it is the same format used for all raw data points and the code input.
+
+# Directory Set-Up
```

Comment:
Its great you included this, a crucial step that can be overlooked by someone just starting with data analysis using programming or learning a new programming language for the first time.